src/CMakeLists.txt

```diff
@@ -2,13 +2,25 @@ add_library(QpSolverCollection
   QpSolverCollection.cpp
   )
 
-target_include_directories(QpSolverCollection PUBLIC
-  ${PROJECT_SOURCE_DIR}/include
-  ${catkin_INCLUDE_DIRS}
-  )
-target_link_libraries(QpSolverCollection PUBLIC
-  ${catkin_LIBRARIES}
+if(QP_SOLVER_COLLECTION_STANDALONE)
+  target_compile_features(QpSolverCollection PUBLIC cxx_std_17)
+  target_link_libraries(QpSolverCollection PUBLIC Eigen3::Eigen)
+
+  install(TARGETS QpSolverCollection
+          EXPORT ${PROJECT_NAME})
+else()
+  target_include_directories(QpSolverCollection PUBLIC
+    ${catkin_INCLUDE_DIRS})
+
+  target_link_libraries(QpSolverCollection PUBLIC
+    ${catkin_LIBRARIES}
   )
+endif()
+
+target_include_directories(QpSolverCollection PUBLIC
```

Comment:
I'm not sure but this part is necessary only for `QP_SOLVER_COLLECTION_STANDALONE`.

Maybe Catkin manages include directory in the following line.
https://github.com/isri-aist/QpSolverCollection/blob/cd6a2490a5ffe76f11e43b015d659a42096e8154/CMakeLists.txt#L65